jsk_pcl_ros/src/particle_filter_tracking_nodelet.cpp

```diff
@@ -394,6 +398,31 @@ namespace jsk_pcl_ros
     frame_id_ = pc.header.frame_id;
     reset_tracking_target_model(new_target_cloud);
   }
+
+  void ParticleFilterTracking::renew_model_with_marker_topic_cb(const visualization_msgs::Marker &marker)
+  {
+    if(marker.type == visualization_msgs::Marker::TRIANGLE_LIST && !marker.points.empty()){
+      ROS_INFO("Reset Tracker Model with renew_model_with_marker_topic_cb");
+      pcl::PointCloud<PointT>::Ptr cloud(new pcl::PointCloud<PointT>);
+      markerMsgToPointCloud(marker,
+                            marker_to_pointcloud_sampling_nums_,
+                            *cloud
+                            );
+
+      Eigen::Affine3d trans_3d; Eigen::Affine3f trans;
+      tf::poseMsgToEigen(marker.pose, trans_3d);
```

Comment:
if you include pcl_conversion_util.h, you can use tf::poseMsgToEigen with Eigen::Affine3f